### DISCUSSION TITLE
Upgrade project not getting bundles css.

### DISCUSSION THREAD

**Discussion by Keyan01** [microsoft/fluentui-blazor#2694] [Q&A] [answered]
I have a project that I am changing the component library to fluent but when I run it does most things correct, but the grid spacing is missing. So I created a new project based on the template in that one I can see the following in the content.
![image](https://github.com/user-attachments/assets/105c7f4f-1707-4840-8f1d-9cfad6eb06eb)
However in my old project I only get this
![image](https://github.com/user-attachments/assets/a50e4a52-b18b-42e6-b054-96a0542cb143)

any pointer as to what is wrong and how to get the content css to load would be appreciated.

**Comment by vnbaaij** [comment, marked as answer]
In the old project, do you have this in your html code: 

```
<link href="{PROJECT_NAME}.styles.css" rel="stylesheet" />
```
Our CSS bundle gets imported through that file.

**Reply by Keyan01** [reply]
Thank you, guessed there was something I had / was missing
